pkg/reconciler/v1alpha1/autoscaling/kpa/kpa.go

```diff
@@ -75,7 +76,7 @@ type KPAScaler interface {
 	Scale(ctx context.Context, pa *pav1alpha1.PodAutoscaler, desiredScale int32) (int32, error)
 
 	// GetScale returns the current scale resource for the PA.
-	GetScale(ctx context.Context, pa *pav1alpha1.PodAutoscaler) (*autoscalingapi.Scale, error)
+	GetScale(pa *pav1alpha1.PodAutoscaler) (*autoscalingapi.Scale, error)
```

Comment:
Why drop `ctx`?

Reply:
No longer required.

Reply:
Should we name this `GetScaleTarget` or something along these lines? `GetScale` sounds a lot like getting the last Scaling decision.

Reply:
I went ahead with `GetScaleResource`.